NEWS.md

```diff
@@ -1,3 +1,12 @@
+MixedModels v4.1.0 Release Notes
+========================
+* Add support for specifying a fixed value of `σ`, the residual standard deviation,
+  in `LinearMixedModel`. `fit` takes a keyword-argument `σ`. For models constructed
+  separately, `σ` can be specified by setting the corresponding field in the `optsum`
```

Comment:
```suggestion
  separately, `σ` can be specified by setting `optsum.sigma`. [#551]
```
The field is called `optsum.sigma` below and the phrasing used here might be interpreted as `optsum.σ`.